content/200-concepts/100-components/01-prisma-schema/08-features-without-psl-equivalent.mdx

```diff
@@ -32,18 +31,18 @@ model User {
 
 There are two reasons to use a database-level function:
 
-* There is no equivalent Prisma function (for example, `gen_random_bytes` in PostgreSQL).
-* You cannot or do not want to rely on functions such `uuid()` and `cuid()`, which are only implemented at Prisma level and do not manifest in the database.
+- There is no equivalent Prisma function (for example, `gen_random_bytes` in PostgreSQL).
+- You cannot or do not want to rely on functions such `uuid()` and `cuid()`, which are only implemented at Prisma level and do not manifest in the database.
 
-    Consider the following example, which sets the `id` field to a randomly generated `UUID`:
+  Consider the following example, which sets the `id` field to a randomly generated `UUID`:
 
-    ```prisma
-    model Post {
-        id    String      @id @default(uuid())
-    }
-    ```
+  ```prisma
+  model Post {
+      id    String      @id @default(uuid())
+  }
+  ```
 
-     The UUID is _only_ generated if you use Prisma Client to create the `Post`. If you create posts in any other way, such as a bulk import script written in plain SQL, you must generate the UUID yourself.
+  The UUID is _only_ generated if you use Prisma Client to create the `Post`. If you create posts in any other way, such as a bulk import script written in plain SQL, you must generate the UUID yourself.
```

Comment:
Does this render correctly with the 2 spaces indentation? need to check manually in preview.

Reply:
Looks ok, it just removed obsolete spaces 
<img width="710" alt="image" src="https://user-images.githubusercontent.com/4734297/124760197-eefb8600-df49-11eb-9010-1f1168404a47.png">

Reply:
Interesting that for _this_ the 2 spaces are enough.